### PR TITLE
feat: adds variant for lead1and1 WIP

### PR DESCRIPTION
**Pull request by c-j-j**
- [ ] TileAA no longer used - can be removed
- [ ] TileAF is now a configurable tile, could potentially be renamed to something like TileDynamicVertical1

### CODE REVIEW

File: packages/section/src/section.tsx [hunk 1]
```diff
@@ -134,7 +134,7 @@ const Section: React.FC<Props> = ({
 
   const data = isPuzzle
     ? createPuzzleData(slices, editionBreakpoint)
-    : prepareSlicesForRender(isTablet)(slices);
+    : prepareSlicesForRender(isTablet, title)(slices);
```
Comment (aronshamash) [inline packages/section/src/section.tsx:137]: `sectionTitle` ?

File: packages/types/index.ts [hunk 2]
```diff
@@ -1,2 +1,3 @@
+export * from "./ui-types";
```
Comment (aronshamash) [inline packages/types/index.ts:1]: 👍

File: packages/section/src/utils/addSliceConfig.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+interface TransformSlice {
+  index: number;
+  name: string;
+  sectionTitle: string;
+  transform: (slice: Slice) => Slice;
+}
+
+const transformLeadOneAndOneNews = {
```
Comment (aronshamash) [inline packages/section/src/utils/addSliceConfig.ts:16]: `transformLeadOneAndOneNews` -> `leadOneAndOneNewsTransform` ?  🤷

File: packages/section/src/utils/addSliceConfig.ts [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  name: "LeadOneAndOneSlice",
+  transform: (slice: Slice) => ({
+    ...slice,
+    support: { ...slice.support, config: { showImage: true } },
+  }),
+};
+
+const sliceTransformations: TransformSlice[] = [transformLeadOneAndOneNews];
+
+export const addSliceConfig = (isTablet: boolean, sectionTitle: string) => (
```
Comment (aronshamash) [inline packages/section/src/utils/addSliceConfig.ts:28]: was thinking `addSliceConfig` might be `transformSlices` ? @Geoff-Ford @justadamhopkins what do you think is clearer

File: packages/section/src/utils/addSliceConfig.ts (hunk 3 above)
Comment (c-j-j) [reply packages/section/src/utils/addSliceConfig.ts:16]: Yes that'll be better

File: packages/section/src/utils/addSliceConfig.ts (hunk 4 above)
Comment (c-j-j) [reply packages/section/src/utils/addSliceConfig.ts:28]: `transformSlices` is a fairly general name, and could result in logic being put here that isn't related to adding these `config` fields. Also, when this is called in the context of  `prepareSlicesForRender` - I think it helps describe what is happening here.

**Review by aronshamash — APPROVED** [review]

**Review by aronshamash — COMMENTED** [review]
looks great

**Review by Geoff-Ford — APPROVED** [review]

**Pull request merged**